src/js/tools/metroconfig.ts

```diff
@@ -0,0 +1,119 @@
+import type { MetroConfig, MixedOutput, Module, ReadOnlyGraph } from 'metro';
+
```

Comment:
Also, if Sentry gets removed after modifying the metro config, will it break the user project?

Reply:
Yes, but that's expected. We tell users to remove all sentry imports after they remove the sentry package.